src/backend/aspen/cli/sync_auth0.py

```diff
@@ -99,6 +116,10 @@ def get_org_members(self, org: Auth0Org) -> List[Auth0User]:
             "members",
         )
 
+    def get_org_invitations(self, org: Auth0Org) -> List[Auth0Invitation]:
+        # Not a paginated endpoint
```

Comment:
are we sure we're sure? I think this is paginated -- https://auth0.com/docs/api/management/v2/#!/Organizations/get_invitations

Reply:
Okay, that's what I assumed at first but `get_all_results` kept giving me strange errors and when I examined the API response there was no pagination info, but I don't doubt I could have made a mistake here (after all why would the docs have page number fields?). I will take a look at it again

Reply:
Okay, I figured out what is going on. For whatever reason this endpoint does not return the usual object structured like `{ 'start': 0, 'limit': 25, 'length': 25, 'keyword': [...] }`. Instead the callable just returns a bare array of invitations 🥴

Reply:
yeah, there's a flag like 'include_totals=true' or something that needs to be passed to the API to tell it to return the extra structure!

Reply:
The library doesn't support it on that call for some reason: https://github.com/auth0/auth0-python/blob/0be88df898dcd1ed0a870d996fefd08b4a06ed12/auth0/v3/management/organizations.py#L341
🥴

Reply:
Yep, sounds good